src/activy/connectStrava.py

```
@@ -1,30 +1,30 @@
-    # the challange button is an imageView which has a varries ammount of instances
-    width = device.info["displayWidth"]
-    height = device.info["displayHeight"]
-    x = int(width * 12 / 13)
-    y = int(100)
-    device.click(x, y)
+    clickSettings(device)
 
     try:
         tryCheckStage(device, "mainMenu", templates)
         raise ValueError("Didn't move to settings.")
     except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)